src/queries.graphql.ts

```diff
@@ -0,0 +1,41 @@
+import { gql } from "./graphql";
+
+export const GET_MARKETPLACE_INTEGRATIONS_TRANSLATIONS = gql`
```

Comment:
Probably not the most critical thing for now, but if anyone ever has more than 100 marketplace offerings, we'll need to paginate.